### DISCUSSION TITLE
Shutdown HaishinKit's Discord channel on December 31, 2024.

### DISCUSSION THREAD

**Discussion by shogo4405** [HaishinKit/HaishinKit.swift#1649] [General]
I will shut down this Discord channel on December 31, 2024.
The reason is that I’ve been feeling fatigued from community support HaishinKit, and I believe it’s not good to push myself any further.

Please understand that I will not be responding to technical questions. Thank you very much for all your support so far.
The content will not be retained, so please save any necessary information on your own.

Thank you.

**Comment by shogo4405** [comment]
GitHub Discussions and Issues is still under consideration. Thank you all for your understanding.

**Comment by alexmck** [comment]
Hi shogo,

I would like to say thank you, and let you know that your work on HaishinKit is very much appreciated. Your attention to detail, and knowledge of video streaming is second to none. You should be very proud of what you have created.

With that said, you need to make sure that your personal health and wellbeing comes first. I completely support your decision to shutdown your Discord server. I too have witnessed an uptick in people recently demanding that you do their homework for them. This very much goes against the spirit of open source software.

I do feel that there is a lot of benefit in still retaining Issues, and to a lesser extent Discussions, but these too I can see are becoming prone to people asking for their homework to be done. At the very least, I would really like to see Issues kept, but please know that I will support your decision whatever you decide to do. I would also like to mention that you should never feel obligated to respond to Issues or Discussions. I would also support you closing Issues and Discussions where people are not acting in the open source spirit, and for serious offenders I would also support you banning these people from partaking in Issues and Discussions.

Either way, whatever you decide, I just want to say thank you. Your push towards the 2.0 release has been really fun to watch. And if you need a break from time to time, please do so. You owe us nothing, and yet you have given us everything. I really appreciate all the hard work and dedication that you have put into HaishinKit. It’s truely amazing.

**Comment by jhays** [comment]
Thank you, Shogo! We really appreciate all your hard work!
